activate-virtualenv.sh

```diff
@@ -25,11 +25,17 @@ if ! type "workon" &>/dev/null; then
   source $virtualenvwrapper_path
 fi
 
+if ! which python2.7 &>/dev/null; then
```

Comment:
what if i have python 2.7 installed but the symlink is /usr/bin/python which is the default?

i think the reason we have a /usr/bin/python2.7 (not really standard) on our dev servers is b/c we can't remove python 2.6.